data/llama_recipe_pipeline.py

```
@@ -0,0 +1,109 @@
+        strTags=row["strTags"],
+        strInstructions=row["strInstructions"],
+        strMealThumb=row["strMealThumb"],
+        strSource=row["strSource"],
+        strYoutube=row["strYoutube"],
+        ingredient_list=ingredient_list
+    )
+
+    response_text = query_llama(user_prompt, SYSTEM_PROMPT)
+    try:
+        data = safe_json_loads(response_text)
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)